src/main/webapp/run-headless_fr.html

```diff
@@ -1,9 +1,9 @@
 <div>
   <p>
     Si le build ne n&eacute,cessite pas un acc&egrave; &agrave; l'environnement graphique (s'il n'utilise que des outils
-    en ligne de commande) cette option peut &ecirc;tre activ�e.
+    en ligne de commande) cette option peut &ecirc;tre activ�e.
```

Comment:
strange change...
maybe fix it using `\uddd` notation

Reply:
Should be good. I think there was a problem with the original file as well, as the `Ž` should have been `&eacute;`.